tests/v2/actions/charts/backuprestore.go

```diff
@@ -0,0 +1,226 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	bv1 "github.com/rancher/backup-restore-operator/pkg/apis/resources.cattle.io/v1"
```

Comment:
this is causing issues in jenkins runs

Reply:
fixed by including the go.mod/sum files in the PR